src/contexts/W3iContext/index.tsx

```diff
@@ -153,13 +125,20 @@ const W3iContextProvider: React.FC<W3iContextProviderProps> = ({ children }) =>
     })
 
     const inviteSentSub = chatClient.observe('chat_invite_sent', { next: refreshChatState })
+    const pingSub = chatClient.observe('chat_ping', {
+      next: () => {
+        console.log('Got a ping!')
+        refreshChatState()
```

Comment:
Can you provide a bit more context on those bits ? Why do we call refreshChatState on chat_ping ?

Reply:
The sync store updates are communicated via chat pings, although this is a bit hacky, it gets the job done and we update state on all other chat events anyway